XenModel/Actions/DelegatedAsyncAction.cs

```diff
@@ -39,31 +39,18 @@
 
 namespace XenAdmin.Actions
 {
-    // An action which just runs a delegate.
-    // It is bad style to use this for more than the very simplest delegate: preferably just a single API call.
-    // Anything more complicated should be turned into a proper action of its own.
-
+    /// <summary>
+    /// An action which just runs a delegate.
+    /// It is bad style to use this for more than the very simplest delegate: preferably just a single API call.
+    /// Anything more complicated should be turned into a proper action of its own.
+    /// </summary>
     public class DelegatedAsyncAction : AsyncAction
     {
         private static readonly log4net.ILog log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
         private readonly Action<Session> invoker;
         private readonly string endDescription;
 
-        public DelegatedAsyncAction(IXenConnection connection, string title, string startDescription, string endDescription,

```

Comment:
The compiler understands these constructors, however the latest Resharper issues warnings, so I removed it as a code style improvement.